### ISSUE TITLE
Playback is not supported with 04.00.02.05 version

### ISSUE THREAD

**Issue by danilofariadutra** [dji-sdk/Mobile-SDK-Android#1252]
After I updated Zenmuse P1 to 04.00.02.05 v, Playback is not supported anymore. Is there any way to use "setCustomExpandFileName" method with this version?

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92985'>Zendesk ticket #92985</a>:<br><br>I would like to know the steps to enable the P1 replay feature. Could you please provide me with the instructions?
P1 may not support setCustomExpandFileName, which means it cannot set the file extension for photos.
<br><p><em><small>°°°</small></em></p>

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92985'>Zendesk ticket #92985</a>:<br><br>I have a question regarding the photos generated after setting up P1. Did you check if they have any suffix?
<br><p><em><small>°°°</small></em></p>

**Comment by danilofariadutra** [comment]
Yes, it does! But, in hardcoded mode everything works, but when I try to input in a formatted string, does not work. The string model will similar as this ""#W5#X2#2#Y3#0#Z4#5"".

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92985'>Zendesk ticket #92985</a>:<br><br>The characters you have used are considered illegal characters. Our camera only supports numbers, letters, Chinese words, and the special symbol "-".
<br><p><em><small>°°°</small></em></p>

**Comment by danilofariadutra** [comment]
So, why using hardcoded mode in code works fine?

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92985'>Zendesk ticket #92985</a>:<br><br>Whether it is hard-coded or formatted strings, the camera will ultimately check if the suffix format is correct. The "#" in "#W5#X2#2#Y3#0#Z4#5" should be an invalid character. Did you use the "#" in the hard-coded way for the photo suffix?
<br><p><em><small>°°°</small></em></p>
